con-abbrevs.json

```diff
@@ -0,0 +1,147 @@
+Concordance
```

Comment:
I think this file can be deleted

Reply:
Also, we MUST make sure the actual verse content from the concordance is not saved anywhere in the app. As it is subject to change if esv updates. We always have to get the content through the API to ensure it's updated